src/mixins/portfolio.js

```diff
@@ -96,6 +97,125 @@ export default {
         order: this.createdOrder,
       });
     },
+    currentOrderBy() {
+      return this.currentTab === 'collected'
+        ? this.collectedOrderBy
+        : this.createdOrderBy;
+    },
+    currentOrder() {
+      return this.currentTab === 'collected'
+        ? this.collectedOrder
+        : this.createdOrder;
+    },
+    currentOrderOptions() {
+      return this.currentTab === 'collected'
+        ? this.collectedOrderOptions
+        : this.createdOrderOptions;
+    },
+    formattedOrderLabel() {
+      let formattedOrderBy = '';
+      switch (this.currentOrderBy) {
+        case ORDER_COLLECTED_CLASS_ID_BY.PRICE:
+        case ORDER_CREATED_CLASS_ID_BY.PRICE:
+          formattedOrderBy = this.$t('order_menu_price');
+          break;
+        case ORDER_COLLECTED_CLASS_ID_BY.LAST_COLLECTED_NFT:
+        case ORDER_CREATED_CLASS_ID_BY.ISCN_TIMESTAMP:
+          formattedOrderBy = this.$t('order_menu_time');
+          break;
+        case ORDER_COLLECTED_CLASS_ID_BY.NFT_OWNED_COUNT:
+          formattedOrderBy = this.$t('order_menu_collected');
+          break;
+        default:
+          break;
+      }
+      return formattedOrderBy;
+    },
+    collectedOrderOptions() {
+      const options = [
+        {
+          value: {
+            orderBy: ORDER_COLLECTED_CLASS_ID_BY.PRICE,
+            order: ORDER.DESC,
+          },
+          name: this.$t('order_menu_price'),
+          format: ORDER_COLLECTED_CLASS_ID_BY.PRICE + ORDER.DESC,
+        },
```

Comment:
I think the value can keep as string like
```js
`${ORDER_CREATED_CLASS_ID_BY.PRICE}-${ORDER.DESC}`
```

so you can get them by `.split('-')`

without adding `format` property in the options